src/pages/SettingsPage.tsx

```diff
@@ -1,7 +1,78 @@
-import React from "react";
+import React, { useState } from "react";
+import { Box, Button } from "@material-ui/core";
 import { Header } from "components/Header";
+import { DialPad } from "components/DialPad";
+import { SettingsRadioGroup } from "components/SettingsRadioGroup";
 import { labels } from "labels";
 
 export const SettingsPage: React.FC = () => {
-  return <Header title={labels.telephonySettings} toPage="about" />;
+  const [currentPhone, setCurrentPhone] = useState("computer");
+  const [currentPhoneOptions] = useState([
+    {
+      value: "computer",
+      label: `${labels.computerVoIP}`,
+    },
+    {
+      value: "officePhone",
+      label: `${labels.officePhone}`,
+    },
+    {
+      value: "otherPhone",
+      label: `${labels.otherPhone}`,
+    },
+  ]);
+  const [forwardYourCalls, setForwardYourCalls] = useState(
+    "cancelCallForwarding"
+  );
+  const [forwardYourCallsOptions] = useState([
+    {
+      value: "cancelCallForwarding",
+      label: `${labels.dontForward}`,
+    },
+    {
+      value: "forwardingToVoicemail",
+      label: `${labels.forwardTo} ${labels.voicemail}`,
+    },
+    {
+      value: "forwardingToOtherPhone",
+      label: `${labels.forwardTo} ${labels.otherPhone}`,
+    },
+  ]);
+  const [number, setNumber] = useState("");
+  const [otherPhoneNumber, setOtherPhoneNumber] = useState("");
+
+  return (
+    <Box>
+      <Header title={labels.telephonySettings} toPage="about" />
+      <SettingsRadioGroup
+        name="currentPhone"
+        title={labels.selectCurrentPhone}
+        subtitle={labels.selectNomadicModeLabel}
+        options={currentPhoneOptions}
+        value={currentPhone}
+        setValue={(to) => setCurrentPhone(to)}
+        otherPhoneNumber={otherPhoneNumber}
```

Comment:
please make this `otherPhoneNumber` & `setOtherPhoneNumber` props optional

This first group do not need to pass these props